### ISSUE TITLE
Proposition 13.147

### ISSUE THREAD

**Issue by Provatiscus** [LaurentClaessens/mazhe#127]
Dans la proposition 13.147 il me semble qu'il faille ajouter la condition que a n'appartient pas à Ai si i > k

**Comment by LaurentClaessens** [comment]
Il n'y a pas de i>k. Il y a seulement un nombre fini (exactement k) d'ensembles A_i

**Comment by Provatiscus** [comment]
Ah oui, j'avais mal lu, mais du coup tu ne traites pas le cas où a appartient seulement à certains Ai. Par exemple le point 0 pour une fct f définie sur ]0,1[ U ]3,4[. Si on note A0 =]0,1[ et A1 = ]3,4[ on n'a pas a € Adh Ai pour tout i inferieur à l.

Je vais essayer de noter mes remarques sur un fichier word et de toutes les envoyer d'un coup car je spams un peu trop je trouve

**Comment by LaurentClaessens** [comment]
Non, elle ne traite pas ce genre de cas, mais ce n'est pas son but. La continuité étant une notion locale, le cas que tu donnes se traite en ne considérant la fonction que sur ]0,1[.

Je vais tenter de reformuler pour être plus clair.
